java-tools/OsmAndMapCreatorUtilities/src/main/java/net/osmand/obf/IconVisibilityComparator.java

```diff
@@ -0,0 +1,176 @@
+package net.osmand.obf;
+
+import net.osmand.PlatformUtil;
+import net.osmand.ResultMatcher;
+import net.osmand.binary.BinaryMapDataObject;
+import net.osmand.binary.BinaryMapIndexReader;
+import net.osmand.render.RenderingRuleSearchRequest;
+import net.osmand.render.RenderingRulesStorage;
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserException;
+
+import java.io.*;
+import java.util.*;
+
+import static net.osmand.binary.BinaryMapIndexReader.*;
+
+public class IconVisibilityComparator {
+	String RENDER_FILE = "/home/user/osmand/issues/i985/default.render.xml";
```

Comment:
Path should be relative and it should be using default render xml and synthtetic. Through parameters or env it could be overridden